datadog_lambda/tracing.py

```diff
@@ -234,7 +234,7 @@ def set_dd_trace_py_root(trace_context, merge_xray_traces):
 
 
 def create_function_execution_span(
-    context, function_name, is_cold_start, trace_context
+    context, function_name, is_cold_start, trace_context, merge_xray_traces
```

Comment:
Don't you need to change the signature where this method is called as well?

Reply:
Whoops, good catch. Looks like I had a file I didn't stage into the commit. Should be updated